### PR TITLE
fixed ser ent tab display

### PR DESCRIPTION
**Pull request by MichaelHabermas**
### Description
fixed thae display tab

**What work was done?**
- Describe in detail what's been modified/created

**Why was this work done?**
- What is the purpose of the feature?

**What feature / user story is it for?**
(Link if possible) 
**Any relevant links or images / screenshots**

### Type of change
Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)

Testing
- Has this feature been tested before conflict?
- Has this feature been tested after conflict?

Change Status
- Completed, ready to review and merge

### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- My code has been reviewed by at least one peer
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- There are no merge conflicts

### CODE REVIEW

**Review by ahmedseragcodes — APPROVED** [review]

**Review by blackenjoy27 — APPROVED** [review]
Nice work!

File: src/components/common/ServicesTable/ServiceTable.js [hunk 1]
```diff
@@ -82,6 +81,7 @@ const ServicesTable = ({
       defaultSortOrder: 'descend',
 
       render: (_, record) => {
+        console.log('record: ', record);
```
Comment (santorap) [inline src/components/common/ServicesTable/ServiceTable.js:84]: Should remove this in a future PR.

File: src/components/common/ServicesTable/ServiceTable.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ import {
   editRecipientAction,
   deleteRecipientAction,
 } from '../../../state/actions';
-import TextArea from 'antd/lib/input/TextArea';
+// import TextArea from 'antd/lib/input/TextArea';
```
Comment (santorap) [inline src/components/common/ServicesTable/ServiceTable.js:33]: If this is not used or needed, it should be deleted.

File: src/index.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ import { LoginPage } from './components/pages/Login';
 import { config } from './utils/oktaConfig';
 import { LoadingOutlined } from '@ant-design/icons';
 import { NavbarHeader } from './components/common/index';
-import { CategoriesPage } from './components/pages/Categories';
+// import { CategoriesPage } from './components/pages/Categories';
```
Comment (santorap) [inline src/index.js:28]: Categories can be deleted outright, right?

File: src/index.js [hunk 4]
```diff
@@ -66,7 +66,7 @@ function App() {
         <SecureRoute path="/profile" component={MyProfile} />
         <SecureRoute path="/employees" component={EmployeesPage} />
         <SecureRoute path="/programs" component={ProgramsPage} />
-        <SecureRoute path="/categories" component={CategoriesPage} />
+        {/* <SecureRoute path="/categories" component={CategoriesPage} /> */}
```
Comment (santorap) [inline src/index.js:69]: Ditto?

**Review by santorap — COMMENTED** [review]
Commenting out these changes works, but it seems like most of those items can be removed directly instead of commenting them out?

**Pull request merged**